Jenkinsfile

```diff
@@ -140,149 +128,92 @@ pipeline {
             }
             post {
-                always {
-                    script {
-                        STAGE = "Package"
-                    }
-                }
                 success {
-                    colourText("info","Packaging Successful!")
+                    colourText("info","Stage: ${env.STAGE} successful!")
                 }
                 failure {
-                    colourText("warn","Something went wrong!")
+                    colourText("warn","Stage: ${env.STAGE} failed!")
                 }
             }
         }
 
-        stage('Deploy'){
+        stage('Deploy CF'){
             agent any
-            when {
-                 expression {
-                     isBranch(BRANCH_DEV) || isBranch(BRANCH_TEST) || isBranch(BRANCH_PROD)
-                 }
+            when{ expression{ isBranch("master") }}
+            environment{ 
+                STAGE = "Deploy CF"
+                DEPLOY_TO = "dev" 
             }
             steps {
-                script {
-                    STAGE = "Deploy"
-                }
                 milestone(1)
                 lock('CH Deployment Initiated') {
-                    colourText("info", "${env.DEPLOY_NAME}-${CH_TABLE}-${MODULE_NAME} deployment in progress")
+                    colourText("info", "${env.DEPLOY_TO}-${CH_TABLE}-${MODULE_NAME} deployment in progress")
                     deploy(CH_TABLE, false)
-                    colourText("success", "${env.DEPLOY_NAME}-${CH_TABLE}-${MODULE_NAME} Deployed.")
+                    colourText("success", "${env.DEPLOY_TO}-${CH_TABLE}-${MODULE_NAME} Deployed.")
                 }
                 lock('VAT Deployment Initiated') {
-                    colourText("info", "${env.DEPLOY_NAME}-${VAT_TABLE}-${MODULE_NAME} deployment in progress")
+                    colourText("info", "${env.DEPLOY_TO}-${VAT_TABLE}-${MODULE_NAME} deployment in progress")
                     deploy(VAT_TABLE, false)
-                    colourText("success", "${env.DEPLOY_NAME}-${VAT_TABLE}-${MODULE_NAME} Deployed.")
+                    colourText("success", "${env.DEPLOY_TO}-${VAT_TABLE}-${MODULE_NAME} Deployed.")
                 }
                 lock('PAYE Deployment Initiated') {
-                    colourText("info", "${env.DEPLOY_NAME}-${PAYE_TABLE}-${MODULE_NAME} deployment in progress")
+                    colourText("info", "${env.DEPLOY_TO}-${PAYE_TABLE}-${MODULE_NAME} deployment in progress")
                     deploy(PAYE_TABLE, false)
-                    colourText("success", "${env.DEPLOY_NAME}-${PAYE_TABLE}-${MODULE_NAME} Deployed.")
+                    colourText("success", "${env.DEPLOY_TO}-${PAYE_TABLE}-${MODULE_NAME} Deployed.")
                 }
-		        lock('Legal Unit Deployment Initiated') {
-                    colourText("info", "${env.DEPLOY_NAME}-${LEU_TABLE}-${MODULE_NAME} deployment in progress")
+                lock('Legal Unit Deployment Initiated') {
```

Comment:
Legal Unit (Guest Appearance)?

Reply:
@ranjit-das asked me to put it back in

Reply:
@willis7 I have realised that I misundrestood it with LEU load available in GitLab. So could you please remove it from here. Sorry for the confusion.